common/src/main/java/com/wynntils/services/chat/ChatTabService.java

```diff
@@ -29,7 +29,7 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public final class ChatTabService extends Service {
-    private final ChatComponent FALLBACK_CHAT = new ChatComponent(McUtils.mc());
```

Comment:
I think the proper fix would have been to make it static, no?

Reply:
idk how you feel about having statics in a service class

Reply:
if you checked and service class, you would find many statics. Same in models